README.md

```diff
@@ -1,389 +1,59 @@
-
-- #### [Watch event changes](https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/blob/main/samples/watch_event.bal)
-
-    This sample shows how to watch for changes to events in an authorized user's calendar. It is a subscription to receive push notification from Google on events changes.  The calendar id and callback url are required to do this operation. Channel live time can be provided via an optional parameter. By default it is 604800 seconds. This operation returns  `WatchResponse` if successful. Else returns `error`.
-
-- #### [Stop channel](https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/blob/main/samples/stop_channel.bal)
-
-    This sample shows how to stop an existing subscription. The channel id and resource is are required to do this operation. This operation returns an error `true` if unsuccessful.
-
-## Listener
-
-- #### [Trigger for new event](https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/blob/main/samples/trigger_create_event.bal)
-
-    This sample shows how to create a trigger on new event. When a new event is occurred, that event details can be captured in this listener.
-
-- #### [Trigger for updated event](https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/blob/main/samples/trigger_update_event.bal)
-
-    This sample shows how to create a trigger on an event update. When a new event is updated, that event details can be captured in this listener.
-
-- #### [Trigger for deleted event](https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/blob/main/samples/trigger_delete_event.bal)
-
-    This sample shows how to create a trigger on cancelled event. When a new event is cancelled, that event details can be captured in this listener.
-
-### Building the Source
-
-Execute the commands below to build from the source after installing required Ballerina version.
-
-1. To clone the repository:
-Clone this repository using the following command:
-```shell
-    git clone https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar.git
-```
-2. To build Java dependency
-```shell script
-    ./gradlew build
-```
-
-3. To build the calendar connector module:
-Run this command from the module-ballerinax-googleapis.calendar root directory:
-```shell script
-    bal build ./calendar
-```
-
-4. To build the calendar connector module without the tests:
-```shell script
-    bal build --skip-tests ./calendar
-```
-
+[Google Calendar](https://developers.google.com/calendar) is a time-management and scheduling calendar service developed by Google. It lets users to organize their schedule and share events with others.
+
+This connector provides the capability to programmatically manage events and calendar and listen the events occurred in the calendar. 
+For more information about configuration and operations, go to the module.
+- [ballerinax/googleapis.calendar](calendar/Module.md)
```

Comment:
Let's have a one sentence description about these modules. For e.g. Google Calendar Module.

Reply:
Added in https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/pull/45/commits/33c24844a5693ce13fa4854db3a209ae2fcfb884